packages/opentelemetry-resources/src/constants.ts

```diff
@@ -98,15 +98,15 @@ export const K8S_RESOURCE = {
 };
 
 /** Attributes describing the telemetry library. */
-export const LIBRARY_RESOURCE = {
+export const TELEMETRY_SDK_RESOURCE = {
   /** The name of the telemetry library. */
-  NAME: 'library.name',
+  NAME: 'telemetry.sdk.name',
```

Comment:
Previous attributes names were aligned to the specs (see: https://github.com/open-telemetry/opentelemetry-specification/blob/948304ad9ddf754ca8308cd51f54c1e8ced5c7d2/specification/data-resource-semantic-conventions.md#library). Could you please help me to point corresponding specs issue/PR?

Reply:
The proposed changes are here: https://github.com/open-telemetry/opentelemetry-specification/pull/494. Let me know if it's too early to adopt them.

Reply:
Found specs issue: https://github.com/open-telemetry/opentelemetry-specification/issues/484, I would have made this change in separate PR because specs PR is still not merged.

Reply:
I can go back to what is currently spec'd. Would you prefer that for this PR?

Reply:
I think it is fine for now, you already made significant efforts to change in multiple places I don't want you to change again. Also, specs PR looks in good shape (3 ppl already approved it).